expression/builtin_string_test.go

```diff
@@ -983,3 +983,27 @@ func (s *testEvaluatorSuite) TestLpad(c *C) {
 		}
 	}
 }
+
+func (s *testEvaluatorSuite) TestQuote(c *C) {
+	defer testleak.AfterTest(c)()
+
+	tbl := []struct {
+		arg interface{}
+		ret interface{}
+	}{
+		{"Don\\'t!", `'Don\'t!'`},
```

Comment:
@XuHuaiyu I use `"Don\\'t!"` instead of `"Don\'t!"` because in Go `"Don\'t!"` is illegal and will cause `unknown escape sequence` in compile time. eg: https://play.golang.org/p/9Slt8mhHzD